### PR TITLE
Remove deprecated features from the manual

### LINKED ISSUE #841: Clean manual and tutorial files of historical and now removed functionality
After the current major cleanup is over we should tidy the manual appropriately, either removing references or reducing references to mention features that were included but unsupported before v3.5. The tutorial files should also be checked over e.g. #824

### PR DESCRIPTION
**Pull request by jtkrogel**
Resolves #841 and #824.  Removes references to Hamiltonian elements that have been recently removed from the source.  Relocates an oddly placed section on pw2qmpack out of the labs area and into the external tools section.  Restores the labs to the "workstation" view for users rather than the specialized environment on Vesta.

### CODE REVIEW

**Comment by prckent** [comment]
OK to test

**Comment by prckent** [comment]
OK with these changes, except perhaps keep the Densities section and add a note that it is TBD? The empty sections are a good reminder to us when browsing that content is needed. Or is there an alternative location to the deleted ones?

File: manual/additional_tools.tex [hunk 1]
```diff
@@ -10,28 +10,41 @@ \section{Initialization Tools}
...
     This utility is written in Fortran90, and is distributed as a patch of the Quantum Espresso 
     source code.  The patch, as well as automated QE download and patch scripts, can be found in 
     \texttt{trunk/external\_codes/quantum\_espresso}.
+
+pw2qmcpack used to be used in serial without problems in small systems and becomes a severe bottleneck for a large system.
+Due to the large energy cutoff in the calculation with pw.x required by the pseudopotentials and the increasing sizes of systems we are interested in, one limitation of QE can be easily reached
+that \texttt{wf\_collect=.true.} results in problems of writing and loading correct plane wave coefficients on disks by pw.x because of the 32bit integer issue. Thus, pw2qmcpack.x fails to convert the orbitals for QMCPACK.
+
+Since Quantum ESPRESSO 5.3.0, the new converter has been fully parallelized to overcome that limitation completely.
+By removing \texttt{wf\_collect=.true.} (by default \texttt{wf\_collect=.false.}), pw.x doesn't collect the whole wave function into individual files for each k point but write one small file for each processor instead.
```
Comment (ye-luo) [inline manual/additional_tools.tex]: QE changed default to true since 6.2 (maybe 6.1)

Comment (ye-luo) [reply manual/additional_tools.tex]: Let me fix the context. I need to update tests also.

Comment (prckent) [reply manual/additional_tools.tex]: @ye-luo Please do a later PR that updates all these docs after the QE 6.3 converter is merged.

**Pull request merged**